src/System.Text.Json/tests/NewtonTests/EnumConverterTests.cs

```diff
@@ -0,0 +1,532 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization;
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public class EnumConverterTests
+    {
+        public class EnumClass
```

Comment:
Why are some test classes/enums (like `EnumClass`) nested inside a particular class that holds the Facts (i.e. `EnumConverterTests`), while others are just at the root namespace (like `DuplicateNameEnum`).

Reply:
Will move all outside